mozetl/symbolication/modules_with_missing_symbols.py

```diff
@@ -167,7 +167,10 @@ def are_symbols_available(debug_file, debug_id):
     for name, (version, debug_id, debug_file), count in top_missing
 ]
 
-subject = "Weekly report of modules with missing symbols in crash reports"
+subject = (
+    "Weekly report of modules with missing symbols in crash reports: %s"
+    % datetime.today().strftime("%Y-%m-%d")
```

Comment:
nit: 
```suggestion
subject = (
    f"Weekly report of modules with missing symbols in crash reports: "
    f"{datetime.today().strftime('%Y-%m-%d')}"
)
```

Reply:
The first f-string doesn't have anything to expand, so it won't pass the linter. I tend not to put function calls in f-strings because of the way f-strings get parsed by the Python parser--it's kind of wild though I think they improved that in 3.12 or 3.13.

I'll adjust it so it's in an intermediary string after you weigh in on whether we should use the `--date` arg value.

Reply:
I switched it to use an f-string.